File: Day234_2025/Q234_GFG.java
import java.util.*;

class Q234_GFG {
    public int median(int[][] mat) {
        List<Integer> val = new ArrayList<>();
        int n = mat.length, m = mat[0].length;

        for (int i = 0; i < n; i++) {
            for (int j = 0; j < m; j++) {
                val.add(mat[i][j]);
            }
        }

        Collections.sort(val);
        return val.get((n * m) / 2);
    }
}

File: Day234_2025/Q234_GFG.py
class Solution:
    def median(self, mat):
        val = []
        n, m = len(mat), len(mat[0])

        for i in range(n):
            for j in range(m):
                val.append(mat[i][j])

        val.sort()
        return val[(n * m) // 2]
